programs/openbook-v2/fuzz/Cargo.toml

```diff
@@ -8,8 +8,8 @@ edition = "2021"
 cargo-fuzz = true
 
 [dependencies]
-anchor-lang = { git = "https://github.com/coral-xyz/anchor.git", rev = "2af9cc669", version = "0.28.0" }
```

Comment:
sorry because maybe it was not clear from my last comment, but neither here (nor in the workspace cargo.toml) you need to specify the anchor patched deps (and you probably don't want to because it could make integrations harder!).

You can leave as your initial commit, and the single place that needs to use this patch will be inside `fuzz/Cargo.toml`, i.e. you just need to replace 
```
[patch.crates-io]
anchor-syn = { path = "../../../3rdparty/anchor/lang/syn" }
```
by 
```
[patch."https://github.com/coral-xyz/anchor.git"]
anchor-syn = { git = "https://github.com/openbook-dex/anchor.git", branch = "v0.28.0_2af9cc66-arbitrary" }
```
and in that case we can get rid off the 3rparty folder & all submodules :)